Formula/precomp.rb

```diff
@@ -0,0 +1,28 @@
+class Precomp < Formula
+  desc "Command-line precompressor to achieve better compression"
+  homepage "http://schnaader.info/precomp.php"
+  url "https://github.com/schnaader/precomp-cpp/archive/v0.4.5.tar.gz"
+  sha256 "39add141554f0186200911ab61838b69f5777956b3dfd37d82bb6923607258dc"
+  head "https://github.com/schnaader/precomp-cpp.git"
+
+  def install
+    # HEAD already has that, but current stable version does not compile with clang without that patch
```

Comment:
The commit itself cannot be simply applied, as it changes 3 other files, too and one of those files fails 3 out of 4 hunks during the patch. Would an `inreplace` be ok in the `stable do` block instead? Otherwise, I would have to patch all commits up until the one in question, so we would end up with a non-stable version.